cns/NetworkContainerContract.go

```diff
@@ -69,6 +69,12 @@ const (
 	Vxlan = "Vxlan"
 )
 
+// Address Types
+const (
+	Default   = "Default"
+	Secondary = "Secondary"
```

Comment:
Did we decided to change the address type from `Multitenant` to `Secondary`?

Reply:
yup